public/components/SelectDataSlot.vue

```diff
@@ -1,6 +1,11 @@
 <template>
   <div class="select-data-slot">
-    <view-type-toggle has-tabs v-model="viewTypeModel" :variables="variables">
+    <view-type-toggle
+      has-tabs
+      v-model="viewTypeModel"
+      :variables="variables"
```

Comment:
Because those two props (variables, and trainingVariables) are in the store, why just not access them in the component itself?

Reply:
That could be a potential addition. However, variables was an existing prop and I opted for keeping the code similar. It could be refactored.

Reply:
I think you are right, maybe this component is meant to be reused on other pages (results, etc.)
So make sure that this prop has a default value or that if missing it will not cause errors.

Reply:
Will do.